src/tree.cc

```diff
@@ -196,7 +196,7 @@ void Tree::GetEditedRange(const Nan::FunctionCallbackInfo<Value> &info) {
 
 void Tree::PrintDotGraph(const Nan::FunctionCallbackInfo<Value> &info) {
   Tree *tree = ObjectWrap::Unwrap<Tree>(info.This());
-  ts_tree_print_dot_graph(tree->tree_, stderr);
+  ts_tree_print_dot_graph(tree->tree_, 2);
```

Comment:
This happens because of updating vendor/tree-sitter:

```
../src/tree.cc:199:3: error: no matching function for call to 'ts_tree_print_dot_graph'
  ts_tree_print_dot_graph(tree->tree_, stderr);
  ^~~~~~~~~~~~~~~~~~~~~~~
../vendor/tree-sitter/lib/include/tree_sitter/api.h:423:6: note: candidate function not viable: no known conversion from 'FILE *' (aka '__sFILE *') to 'int' for 2nd argument
void ts_tree_print_dot_graph(const TSTree *, int file_descriptor);
     ^
```

introduced in https://github.com/tree-sitter/tree-sitter/commit/97fd990822deeb3c288f4999a8410fba69f230b6